windows/src/engine/kmcomapi/processes/keyboard/kpinstallkeyboardlanguageprofiles.pas

```diff
@@ -359,6 +345,22 @@ function TKPInstallKeyboardLanguageProfiles.RegisterLocale(KeyboardName, BCP47Ta
     end;
   end;
 
+  { Register the keyboard profile }
```

Comment:
Changing the order of these operations prevented the system keyboard being installed: this is not documented by MSDN but makes sense, I guess.